src/Codeception/Module/Facebook.php

```diff
@@ -121,10 +121,10 @@ public function _requires()
 
     public function _depends()
     {
-        return ['Codeception\Module\PhpBrowser' => $this->dependencyMessage];
+        return ['Codeception\Module\WebDriver' => $this->dependencyMessage];
```

Comment:
Please use `Codeception\Lib\Interfaces\Web`, as it will match both PhpBrowser and WebDriver